app/helpers/comments_helper.rb

```diff
@@ -286,6 +286,17 @@ def tag_comment_as_spam_link(comment)
     end
   end
 
+  # gets the css user-<id> class name for the comment
+  def get_comment_user_css_class(comment)
+    if comment.pseud.nil?
+      "visitor"
+    elsif comment.pseud.user.is_author_of?(comment.ultimate_parent) && comment.ultimate_parent.anonymous?
+      "anonymous-creator"
+    else
+      comment.pseud.user.id
+    end
+  end
+
```

Comment:
You can skip the special cases for anonymous and guests. We just want to add the user ID like we did with work blurbs: 

https://github.com/otwcode/otwarchive/blob/808b34b1d21be65c26fc9ae4a20621c40955a37d/app/helpers/application_helper.rb#L588-L599